src/Compilers/MinifyCompiler.php

```diff
@@ -30,18 +30,27 @@ class MinifyCompiler extends BladeCompiler
     protected $blade;
 
     /**
+     * The ignored paths.
+     *
+     * @var string
+     */
+    protected $ignoredPaths;
+
+    /**
      * Create a new instance.
      *
      * @param \GrahamCampbell\HTMLMin\Minifiers\BladeMinifier $blade
+     * @param array                                           $ignoredPaths
```

Comment:
Please add this to the end of the param list, and default it to `[]` to do the best we can with BC.